packages/manager/src/features/CloudPulse/Widget/components/CloudPulseIntervalSelect.tsx

```diff
@@ -39,7 +45,7 @@ export const getInSeconds = (interval: string) => {
 };
 
 // Intervals must be in ascending order here
-export const all_interval_options = [
+export const all_interval_options: IntervalOptions[] = [
```

Comment:
```suggestion
export const allIntervalOptions: IntervalOptions[] = [
```
We use camel case for variable names throughout the codebase. Can we be consistent?